### ISSUE TITLE
Make monitoring of node groups configurable

### ISSUE THREAD

**Issue by maxemann96** [pulumi/pulumi-eks#1032]
## Hello!

- Vote on this issue by adding a 👍 reaction
- If you want to implement this feature, comment to let us know (we'll work with you on design, scheduling, etc.)

## Issue details
Currently it's not possible to alter the [detailed logging switch](https://www.pulumi.com/registry/packages/aws/api-docs/ec2/launchconfiguration/#enablemonitoring_nodejs) in the underlying ec2 LaunchConfiguration / NodeLaunchTemplate, while managing the resource with this provider. Since we are monitoring our cluster with the Prometheus stack, we don't need the cloudwatch monitoring.

### Affected area/feature
For Node.js, a change should be implemented starting [here](https://github.com/pulumi/pulumi-eks/blob/a7375eed2bef67819a530077193bf37aa35936ac/nodejs/eks/nodegroup.ts#L777)

**Comment by mjeffryes** [comment]
Thanks for filing this @maxemann96. In the meantime, you might be able to manage this using [transformations](https://www.pulumi.com/docs/concepts/options/transformations/)

**Comment by maxemann96** [comment]
I forgot about transformations. Thanks for the info. Will use that as workaround.

**Comment by maxemann96** [comment]
Thanks for the implementation :)
